dts/bindings/serial/intel,lw_uart.yaml

```diff
@@ -17,3 +17,12 @@ properties:
     type: boolean
     description: |
       Baud rate cannot be changed by software (Divisor register is not writable)
+
+  parity:
+    default: "none"
+
+  stop-bits:
+    default: "1"
+
+  data-bits:
+    default: 8
```

Comment:
Seems like you forgot to add the description here?

Reply:
My bad, fixed.